### ISSUE TITLE
packages

### ISSUE THREAD

**Issue by Rokinluohhh** [cvlab-stonybrook/PromptMIL#3]
Can you provide the version of the packages(such as timm)? Looking forward to your reply!

**Comment by jingweizhang-xyz** [comment]
We provided the version of pytorch-lightning related packages (timm==0.6.12), and the rest does not matter. If you encounter any problems, please let me know.

**Comment by Rokinluohhh** [comment]
Thanks for your reply!Have you stained and normalised the patches?

**Comment by jingweizhang-xyz** [comment]
No.

**Comment by Rokinluohhh** [comment]
Hi,can you provide your code for creating patches?

**Comment by jingweizhang-xyz** [comment]
The patches come from a previous project from our co-authors around two years ago. I will contact them to see if they still have that code.

**Comment by Rokinluohhh** [comment]
ok,thans very much.And may I ask if you used cross validation?

**Comment by jingweizhang-xyz** [comment]
No, we use fold=0 as the validation set and repeat the experiment 3 times.

**Comment by Rokinluohhh** [comment]
ok Thanks for your patient reply!

**Comment by Rokinluohhh** [comment]
Hi,why you reshape there?
![微信截图_20240118092851](https://github.com/cvlab-stonybrook/PromptMIL/assets/131835675/948979a1-a831-4d4b-965f-0660abfd7780)

**Comment by jingweizhang-xyz** [comment]
Because we use merged images. For 200 merged images of 224x 224, it shape is h=224*200, w=224, c=3. This reshaping operation convert it back to (200, 224, 224, 3).

**Comment by Rokinluohhh** [comment]
So,I should change this code if i use images that havent merged?

**Comment by jingweizhang-xyz** [comment]
No, you do not need to. This code is capable with images not being merged. The reshaping operation will result in (1, 224, 224, 3).

**Comment by Rokinluohhh** [comment]
Thanks a lot!
